css/types/length.json

```diff
@@ -0,0 +1,881 @@
+{
+  "css": {
+    "types": {
+      "length": {
```

Comment:
Will "length" as a key name work when turned into a JavaScript object, or will it collide with [Object's length property](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)?

Reply:
Good thought, but [JavaScript objects do not have a length property, only Arrays do](https://stackoverflow.com/questions/4690520/javascript-object-literal-length-undefined).

Reply:
MDN Object docs may need an update then...

I tried it out in console, it appears to work, even if it breaks GitHub's syntax highlighter:

```javascript
testJSON = '{"item1": {"prop1": "value1"}, "length": {"prop2": "value2"}}';
obj = JSON.parse(testJSON);
obj.item1  // Object { prop1: "value1" }
obj.length // Object { prop2: "value2" }
```

Reply:
Is there a conventional JavaScript way to avoid clashing with a common name (e.g., in Python there's a convention of appending `_` so `class` becomes `class_`)? If we need to rename the key, then I'd like to avoid doing something too cryptic.

Reply:
I'm wrong again - the ``Object`` constructor has the length parameter, the objects don't. JavaScript.

```javascript
Object.length  // 1
Object().length // undefined
```

@ddbeck I thought we'd hit that issue when we got to length. I'm happy to have a little syntax highlighting weirdness for now, and only worry when it becomes a blocker.

Reply:
@jwhitlock OK, got it. Thank you for thinking of this and taking the time to figure it out!